### DISCUSSION TITLE
Discourse

### DISCUSSION THREAD

**Discussion by dr460nf1r3** [catppuccin/catppuccin#2444] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

Discourse supports color schemes, that can be shared: https://meta.discourse.org/t/create-and-share-a-color-scheme/61196

### Any additional comments?

Discourse is a very widely used Forum software. Having the Catppuccin palette available as a preset would be an awesome thing :) 

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by uncenter** [comment]
See also: https://meta.discourse.org/t/reference-for-full-color-customization/255158/2.

It looks like we can actually turn potential Discourse themes into userstyles as well if the forum admin doesn't add it themselves.

**Comment by dr460nf1r3** [comment]
In the meantime, I went ahead and took the chance to implement it myself using a combination of CSS overrides and other techniques Discourse offers for theming: https://github.com/dr460nf1r3/discourse-catppuccin

It is quite messy CSS, but it works well though 😄

**Comment by WalkQuackBack** [comment]
I made a userstyle for Roblox DevForum which runs a forked version of Discourse. It might be worthwhile to look to see if any variables can be ported back. https://github.com/catppuccin/userstyles/pull/1963

**Reply by ninetailedtori** [reply]
Much of it could be ported, outside of branding-specific changes! Branding-specific overrides will have to be defined under similar to something like https://github.com/catppuccin/userstyles/pull/2116.
